web/src/app/admin/AdminFieldComponents.tsx

```diff
@@ -6,8 +6,10 @@ import InputAdornment from '@material-ui/core/InputAdornment'
 import Switch from '@material-ui/core/Switch'
 import Visibility from '@material-ui/icons/Visibility'
 import VisibilityOff from '@material-ui/icons/VisibilityOff'
+import TelTextField from '../util/TelTextField'
 
 interface InputProps {
+  configtype: string
```

Comment:
Just name this `type` to match the HTML attribute, make it optional with a fixed value of `tel` (since that's the only supported one ATM).